segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/BookKeeperServiceRunner.java

```diff
@@ -169,4 +190,32 @@ private void cleanupDirectories() throws IOException {
     }
 
     //endregion
+
+    public static void main(String[] args) throws Exception {
```

Comment:
It's a little weird to add a main to a class that didn't used to have one and looks to be used already. Why is this here and not in a new file?